packages/docs/pages/internals/components.mdx

```diff
@@ -274,6 +274,75 @@ const movieDB = await create({
 })
 ```
 
+### `sort`
```

Comment:
I know this is gonna be big, but I would call this `sorter`. The verb alone makes little sense.